src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -175,7 +175,6 @@ const StyledPanel = styled(Panel)`
   display: grid;
   grid-template-columns: max-content auto max-content;
   grid-gap: ${space(3)};
-  align-items: center;
```

Comment:
hmm isn't this what we want instead of `align-content`?

Reply:
Actually I wanted to align this to the top similar to our alerts :)